### PR TITLE
more org mapping fixes

### PR DESCRIPTION
**Pull request by teragopher**

### CODE REVIEW

File: map_org_names.yaml [hunk 1]
```diff
@@ -1,2670 +1,3024 @@
...
- - ['^[[:space:]]*zte[[:space:]]corporation[[:space:]]*$','ZTE Corporation']
- - ['^[[:space:]]*zulily[[:space:]]*$','Zulily, Inc.']
- - ['^[[:space:]]*zuora[[:space:]]*$','Zuora Inc.']
- - ['^[[:space:]]*zynga,?[[:space:]]inc\\.?[[:space:]]*$','Zynga, Inc.']
- - ['^[[:space:]]*école[[:space:]]de[[:space:]]technologie[[:space:]]supérieure[[:space:]]*$','École de technologie supérieure (ÉTS)']
- - ['^[[:space:]]*école[[:space:]]pour[[:space:]]*$',"École pour l'Informatique et les Techniques Av"]
- - ['^[[:space:]]*école[[:space:]]de[[:space:]]technologie[[:space:]]supérieure[[:space:]](éts)[[:space:]]*$','ÉTS - École de technologie supérieure']
- - ['^[[:space:]]*нгс[[:space:]]*$','НГС.ТЕХНОЛОГИИ (NGS.RU)']
- - ['^[[:space:]]*horizon[[:space:]]robotics[[:space:]]*$','地平线｜边缘人工智能芯片全球领导者']
+  - ['^[[:space:]]*(1&1|1&1[[:space:]]ionos[[:space:]]cloud)[[:space:]]*$','1&1 IONOS']
```
Comment (lukaszgryglicki) [inline map_org_names.yaml:4]: Indenting, otherwise too big to review - I'll merge run and we will see what happens
/lgtm

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

File: map_org_names.yaml (hunk 1 above)
Comment (teragopher) [reply map_org_names.yaml:4]: @lukaszgryglicki thanks, i just noticed the indenting and put the wip label.. lol . I have fixed that and creating another PR

**Comment by lukaszgryglicki** [comment]
```
{"level":"warning","msg":"select id, name from organizations where name regexp ? and name != ?","time":"2020-11-16T14:41:23Z"}
{"level":"warning","msg":"[1:^[[:space:]]*(erth[[:space:]](centre[[:space:]]for[[:space:]]research[[:space:]]and[[:space:]]technology[[:space:]]hellas)|erth[[:space:]](centre[[:space:]]for[[:space:]]research[[:space:]]and[[:space:]]technology[[:space:]]hellas|erth[[:space:]]\\(?centre[[:space:]]for[[:space:]]research[[:space:]]and[[:space:]]technology[[:space:]]hellas\\)?)[[:space:]]*$ 2:CERTH (Centre for Research and Technology Hellas) ]","time":"2020-11-16T14:41:23Z"}
(...)
8832), Deleted organization 'CapTech Ventures, Inc' (id=42993), Added organization 'CapTech Ventures, Inc' (id=46351),  err:QueryDB: Error 1139: Got error 'missing ) at offset 357' from regexp","time":"2020-11-16T14:41:23Z"}
{"level":"info","msg":"PutMapOrgNames(exit): apiName:PutMapOrgNames username:internal-api-user status: err:PutMapOrgNames: QueryDB: Error 1139: Got error 'missing ) at offset 357' from regexp","time":"2020-11-16T14:41:23Z"}
```

Gotta go for my kid's training, please try to fix and I'll reiterate this in about 2.5-3 hours.

**Pull request merged**